src/main/webapp/populateData.js

```diff
@@ -311,3 +312,60 @@ function populateGo() {
       });
 }
 
+/**
+ * Set up the assign panel. For now, this just prints the response
+ * from the server for /gmail-actionable-emails
+ */
+function setUpAssign() {
+  const assignContent = document.querySelector('#assign');
+
+  const subjectLinePhrases = ['Action Required', 'Action Requested'];
+  fetchActionableEmails(subjectLinePhrases, true, 7)
+      .then((response) => {
+        assignContent.innerText = response
+            .map((obj) => obj.snippet)
+            .reduce((a, b) => a + '\n' + b);
+      })
+      .catch((e) => {
+        console.log(e);
+        if (e instanceof AuthenticationError) {
+          signOut();
+        }
+      });
+}
+
+/**
+ * Get actionable emails from server. Used for assign panel
+ *
+ * @param {string[]} listOfPhrases list of words/phrases that the subject line
+ *     of user's emails should be queried for
+ * @param {boolean} unreadOnly true if only unread emails should be returned,
+ *     false otherwise
+ * @param {number} nDays number of days to check unread emails for.
+ *     Should be an integer > 0
+ * @return {Promise<Object>} returns promise that returns the JSON response
+ *     from client. Should be list of Gmail Message Objects. Will throw
+ *     AuthenticationError in the case of a 403, or generic Error in
+ *     case of other error code
+ */
+function fetchActionableEmails(listOfPhrases, unreadOnly, nDays) {
+  const listOfPhrasesString = encodeListForUrl(listOfPhrases, true);
+  const unreadOnlyString = unreadOnly.toString();
+  const nDaysString = nDays.toString();
+
+  const queryString =
+      `/gmail-actionable-emails?subjectLinePhrases=${listOfPhrasesString}` +
+      `&unreadOnly=${unreadOnlyString}&nDays=${nDaysString}`;
+
+  return fetch(queryString)
+      .then((response) => {
+        switch (response.status) {
+          case 200:
+            return response.json();
+          case 403:
+            throw new AuthenticationError();
+          default:
+            throw new Error(response.status + ' ' + response.statusText);
```

Comment:
I can, but this is used elsewhere in the script file. For consistency, I think this should stay the same for now. 

Plus this is a rather simple string concatenation, so it probably isn't prohibited under style guide (it only talks about [complicated string concatenation being less preferred than template literals](https://google.github.io/styleguide/jsguide.html#features-strings-template-strings).

Reply:
Okay sure, it's definitely not an important change rather it was just in case you wanted to do the same as in line 357. Thanks for addressing my comment!